src/main/java/morgana/task/TaskList.java

```diff
@@ -0,0 +1,64 @@
+package morgana.task;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents a list of tasks, providing methods to add, remove, and
+ * retrieve tasks from the list.
+ */
+public class TaskList {
```

Comment:
Would it be better if the method names were more descriptive eg. addTask, getTask, removeTask, instead of simply add, get, remove?